spec/models/user_spec.rb

```diff
@@ -44,7 +44,73 @@
 
     expect(user.items.count).to eq(3)
     expect(user.open_items.count).to eq(2)
+  end
+
+  it "cannot find won auctions if the user has no bidders" do
+      user = create(:user)
+      expect(user.bids).to be_blank
+      expect(user.won_items.count).to eq(0)
+  end
+
+  it "can identify a single auction won by a specific user" do
+    user = create(:user_with_bids)
+
+    item1, item2, item3 = user.items
 
+    dup_bid = create(:bid, price: 1000, item: item1, user: user)
```

Comment:
Erin and I added a duplicate bid to these tests, when adding the "get distinct" item in the User model.  perhaps this could be it's own test?